pkg/nuclide-clang-rpc/lib/ClangFlagsManager.js

```diff
@@ -90,19 +97,22 @@ export default class ClangFlagsManager {
   _compilationDatabases: Map<string, Map<string, ClangFlags>>;
   _realpathCache: Object;
   _pathToFlags: Map<string, Promise<?ClangFlags>>;
+  _clangProjectFlags: ?ClangProjectFlags;
```

Comment:
good to know.  I didn't realize how that worked, but of course that makes sense.